src/main/java/org/wildfly/security/evidence/BearerTokenEvidence.java

```diff
@@ -0,0 +1,21 @@
+package org.wildfly.security.evidence;
+
+import org.wildfly.common.Assert;
+
+/**
+ * An {@link Evidence} that usually holds a bearer security token.
+ *
+ * @author <a href="mailto:psilva@redhat.com">Pedro Igor</a>
+ */
+public class BearerTokenEvidence implements Evidence {
```

Comment:
Should this be final?  In the realm you make a check that the class types are equal so probably better to not allow extension of this one.